proc/variables.go

```diff
@@ -241,21 +290,69 @@ func parseG(thread *Thread, gaddr uint64, deref bool) (*G, error) {
 	return g, nil
 }
 
+// Returns the address of the named variable
+func (scope *EvalScope) EvalVariableAddr(name string) (*Variable, error) {
```

Comment:
Maybe rename this to `ExtractVariableInfo` since it returns an initialized variable, with all info except for the actual value.